blaze/utils.py

```diff
@@ -262,6 +278,27 @@ def register(typename, converter, converters=_converters):
 object_hook.register('datetime', pd.Timestamp)
 object_hook.register('frozenset', frozenset)
 object_hook.register('datashape', dshape)
+object_hook.register('builtin_function', lambda x: getattr(builtins, x))
+
+
+def numpy_pandas_function_from_str(f):
+    """
+    reconstruct function from string representation
+    """
+    if f.startswith("numpy"):
+        mod = np
+
+    elif f.startswith("pandas"):
+        mod = pd
+
+    else:
+        raise NotImplementedError("accepts numpy/pandas/builtin funcs only")
```

Comment:
How about a [flask NotImplemented exception (501 Not Implemented)](werkzeug.exceptions.NotImplemented)